pymove/core/dataframe.py

```diff
@@ -1802,6 +1821,16 @@ def drop(self, labels=None, axis=0, index=None, columns=None, level=None, inplac
         ----------
         https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.drop.html
         """
+        
+        if type(labels) == str:
+            _labels = set([labels])
+        else:
+            _labels = set(labels)
+     
+        _columns = set(['lat', 'lon', 'datetime'])
+        if inplace and (axis==1 or axis=='columns') and (_labels & _columns):
+            raise AttributeError("Could not drop columns lat, lon, and datetime.")
+     
```

Comment:
Deve permitir ao usuário remover colunas não essenciais se operação não for `inplace`